src/main/kotlin/org/kamiblue/client/gui/rgui/windows/BlockPicker.kt

```diff
@@ -0,0 +1,32 @@
+package org.kamiblue.client.gui.rgui.windows
+
+import net.minecraft.block.Block
+import net.minecraft.util.ResourceLocation
+import org.kamiblue.client.gui.rgui.component.BlockButton
+import org.kamiblue.client.setting.settings.impl.other.BlockSetting
+
+object BlockPicker : ListWindow("Block Picker", 0.0f, 0.0f, 200.0f, 200.0f, SettingGroup.NONE) {
+
+    var setting: BlockSetting? = null
+
+    override var lastActiveTime: Long = Long.MAX_VALUE // Dodgy thing to create always on top
+        set(value) {
+            field = Long.MAX_VALUE
+        }
```

Comment:
I don't see this ending well with multiple block pickers opened at the same time, or having anything else that also overrides it to max value

Reply:
There can never be multiple open at the same time, do you have any suggestions in a different way, otherwise it just appears underneath all of the windows.